code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -137,7 +137,7 @@
 
 /datum/crafting_recipe/improvised_pneumatic_cannon //Pretty easy to obtain but
 	name = "Pneumatic Cannon"
-	result = /obj/item/pneumatic_cannon/ghetto
+	result = list(/obj/item/pneumatic_cannon/ghetto)
```

Comment:
This changes a lot. Please also add a unit test (See `code/modules/unit_test`) to ensure that every subtype of `/datum/crafting_recipe` has result as a list. `islist()` will help here. 

The amount of unknown runtimes this could cause if someone isnt aware is worrying.

Reply:
Could this not just be made into an `ASSERT()` on `/datum/crafting_recipe`?
```dm
/datum/crafting_recipe/New()
	. = ..()
	ASSERT(islist(result))
```

Reply:
These dont get instanced until they are actually being crafted, which doesnt solve the problem. A unit test fires in the CI stage before the PR is even merged, saving bad code from getting in.